public/stylesheets/videoondemand-module.scss

```diff
@@ -0,0 +1,67 @@
+#videoondemand-module {
+    margin: 10px;
```

Comment:
@JonathanACG I think all modules will use this setting. Could you please assign this property to class `.module` in `sytle.scss` please?